### ISSUE TITLE
Remove the CI infrastructure made for the monorepo.

### ISSUE THREAD

**Issue by mythmon** [mozilla/normandy#1188]
Currently we have a complicated CI infrastructure for running various steps in multiple projects. With the monorepo being broken up, this is no longer needed. It is causing complexity in CI, linting, and preventing us from using pre-commit hooks. We should get rid of it.

**Comment by mythmon** [comment]
Blocked by #1189
